# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libpathrs/hotfix-install.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libpathrs-0.2.6/install.sh.vanilla	2026-09-27 00:19:14.690851992 +0300
+++ libpathrs-0.2.6/install.sh	2026-09-27 00:26:04.098805759 +0300
@@ -70,7 +70,7 @@
 
 TARGET=
 # Try to emulate autoconf's basic flags.
-DEFAULT_PREFIX=/usr/local
+DEFAULT_PREFIX=/usr
 usage() {
 	[ "$#" -eq 0 ] || echo "ERROR:" "$@" >&2
 
@@ -245,15 +245,15 @@
 echo "[install] installing libpathrs into DESTDIR=${DESTDIR:-/}"
 set -x
 # pkg-config.
-install -Dt "$DESTDIR/$pkgconfigdir/" -m 0644 pathrs.pc
-install -Dt "$DESTDIR/$includedir/"   -m 0644 include/pathrs.h
+install -D -m 0644 pathrs.pc "$DESTDIR/$pkgconfigdir/"
+install -D -m 0644 include/pathrs.h "$DESTDIR/$includedir/"
 # Static library.
 if [ -n "$install_static" ]; then
-	install -Dt "$DESTDIR/$libdir" -m 0644 "$builddir/libpathrs.a"
+	install -D -m 0644 "$builddir/libpathrs.a" "$DESTDIR/$libdir"
 fi
 # Shared library.
 if [ -n "$install_shared" ]; then
-	install -DT -m 0755 "$builddir/libpathrs.so" "$DESTDIR/$libdir/$SONAME"
+	install -D -m 0755 "$builddir/libpathrs.so" "$DESTDIR/$libdir/$SONAME"
 	ln -sf "$SONAME" "$DESTDIR/$libdir/libpathrs.so.$SOVERSION"
 	ln -sf "$SONAME" "$DESTDIR/$libdir/libpathrs.so"
 fi
